cmd/ui/src/components/SettingsMenu.tsx

```diff
@@ -74,6 +81,12 @@ const SettingsMenu: React.FC<Props> = ({ anchorEl, handleClose }) => {
         dispatch(logout());
     };
 
+    const toggleDarkMode: React.MouseEventHandler<HTMLLIElement> = () => {
+        const theme = darkMode ? 'light' : 'dark';
+        body.setAttribute('class', theme);
+        dispatch(setDarkMode(!darkMode));
```

Comment:
It looks like we're storing information about whether dark mode is enabled in a couple location - as a piece of state in redux, as well as an attribute on the top level <body> tag which isn't controlled by react.

Curious why both are required?

Reply:
We need to set the dark mode class in the body for the Doodle UI elements that we are starting to use. When we ultimately move from MUI, we should only need the class in the DOM to inform the theme. 

Setting the `dark` class somewhere like the element that has the id of `root` does not affect elements like dialog components and thus `tertiary` variant styles do not adapt accordingly in dark mode.

Reply:
Interesting - I see the explanation in the tailwind docs now.

I suggest we find a top level div in `App.tsx` and control a className prop there (using react/redux patterns) rather than reach outside the react root node and manually manipulate the DOM. That way we keep the theme state in one place and don't have to do anything outside what react renders/handles for us.

Reply:
I took care of this in !721